cmd/init.go

```diff
@@ -48,7 +44,7 @@ func newInitCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "init",
 		Short: "Initialize a new project",
-		Long: `Initialize a new project including vendor/ directory and Go package directories.
+		Long: `Initialize a new project including Go package directories.
```

Comment:
The rest of the sentence gives no information at all.
```suggestion
		Long: `Initialize a new project.
```